app/templates/head.hbs

```diff
@@ -6,3 +6,4 @@
     {{/each}}
   {{/if}}
 </title>
+<link rel="stylesheet" href="/school-theming/css/school.css"> 
```

Comment:
To test this change when proxying from your local system, prefix the href with 'https://ilios3-demo.ucsf.edu' so the line reads:

```<link rel="stylesheet" href="https://ilios3-demo.ucsf.edu/school-theming/css/school.css">```